src/app/modules/users/components/users-list/users-list.component.ts

```diff
@@ -18,19 +20,41 @@ export class UsersListComponent implements OnInit, OnDestroy, AfterViewInit {
   dtTrigger: Subject<any> = new Subject();
   @ViewChild(DataTableDirective, { static: false })
   dtElement: DataTableDirective;
+  dtOptions: any = {};
+  isFlagOn;
 
-  constructor(private store: Store<User>, private actionsSubject$: ActionsSubject) {
+  constructor(
+    private store: Store<User>,
+    private actionsSubject$: ActionsSubject,
+    private featureManagerService: FeatureManagerService
+  ) {
     this.isLoading$ = store.pipe(delay(0), select(getIsLoading));
   }
 
   ngOnInit(): void {
+    this.isFeatureToggleActivated().subscribe((flag) => {
+      this.isFlagOn = flag;
+    });
+    this.store.dispatch(new LoadUsers());
     this.loadUsersSubscription = this.actionsSubject$
       .pipe(filter((action: any) => action.type === UserActionTypes.LOAD_USERS_SUCCESS))
       .subscribe((action) => {
         this.users = action.payload;
         this.rerenderDataTable();
       });
-    this.store.dispatch(new LoadUsers());
+
+    this.loadUsersSubscription = this.actionsSubject$
```

Comment:
Not sure if this affects somehow but, It is ok to overwrite the subscription ?. Line 39 and 46.

Reply:
I have changed to a new Subscription.